projects/igniteui-angular/src/lib/chips/chips-area.spec.ts

```diff
@@ -135,459 +130,324 @@ describe('IgxChipsArea', () => {
+            fix.detectChanges();
 
-        firstChip.animateOnRelease = false;
-        secondChip.animateOnRelease = false;
+            expect(chipAreaComp.onSelection.emit).toHaveBeenCalledWith({
+                originalEvent: keyEvent,
+                owner: chipAreaComp,
+                newSelection: [secondChipComp]
+            });
 
-        const firstChipElem = firstChip.chipArea.nativeElement;
-        const secondChipElem = secondChip.chipArea.nativeElement;
+            let chipsSelectionStates = fix.componentInstance.chips.toArray().filter(c => c.selected);
+            expect(chipsSelectionStates.length).toEqual(1);
+            expect(secondChipComp.selected).toBeTruthy();
 
-        const firstChipTop = firstChipElem.getBoundingClientRect().top;
-        const firstChipLeft = firstChipElem.getBoundingClientRect().left;
-        const firstChipBottom = firstChipElem.getBoundingClientRect().bottom;
-        const firstChipRight = firstChipElem.getBoundingClientRect().right;
+            secondChipComp.onChipKeyDown(keyEvent);
+            fix.detectChanges();
 
-        const firstChipX = (firstChipLeft + firstChipRight) / 2;
-        const firstChipY = (firstChipTop + firstChipBottom) / 2;
+            expect(chipAreaComp.onSelection.emit).toHaveBeenCalledWith({
+                originalEvent: keyEvent,
+                owner: chipAreaComp,
+                newSelection: []
+            });
 
-        const secondChipTop = secondChipElem.getBoundingClientRect().top;
-        const secondChipLeft = secondChipElem.getBoundingClientRect().left;
-        const secondChipBottom = secondChipElem.getBoundingClientRect().bottom;
-        const secondChipRight = secondChipElem.getBoundingClientRect().right;
+            chipsSelectionStates = fix.componentInstance.chips.toArray().filter(c => c.selected);
+            expect(chipsSelectionStates.length).toEqual(0);
+            expect(secondChipComp.selected).not.toBeTruthy();
+        });
 
-        const secondChipX = (secondChipLeft + secondChipRight) / 2;
-        const secondChipY = (secondChipTop + secondChipBottom) / 2;
+        it('should be able to have multiple chips selected', () => {
+            fix = TestBed.createComponent(TestChipComponent);
+            fix.detectChanges();
 
-        UIInteractions.simulatePointerEvent('pointerdown', secondChipElem, secondChipX, secondChipY);
-        fix.detectChanges();
+            let selChips = 0;
+            const chipAreaComponent = fix.componentInstance;
 
-        fix.whenStable().then(() => {
+            chipAreaComponent.chipList.push({ id: 'Town', text: 'Town', removable: true, selectable: true, draggable: true });
             fix.detectChanges();
-            UIInteractions.simulatePointerEvent('pointermove', secondChipElem, secondChipX + 10, secondChipY + 10);
 
-            return fix.whenStable();
-        }).then(() => {
+            spyOn(chipAreaComponent.chipsArea.onSelection, `emit`).and.callFake(function() {
```

Comment:
can you change this to  spyOn(chipAreaComponent.chipsArea.onSelection, `emit`) and to check the calls with  expect(chipAreaComponent.chipsArea.onSelection.emit).toHaveBeenCalledTimes(2)